src/main/java/software/amazon/cloudformation/proxy/AmazonWebServicesClientProxy.java

```diff
@@ -110,6 +122,70 @@ public ClientT client() {
         };
     }
 
+    public <ClientT, ModelT, CallbackT extends StdCallbackContext>
+        Initiator<ClientT, ModelT, CallbackT>
+        newInitiator(@Nonnull Supplier<ClientT> client, final ModelT model, final CallbackT context) {
+        return newInitiator(newProxy(client), model, context);
+    }
+
+    @Override
+    public <ClientT, ModelT, CallbackT extends StdCallbackContext>
```

Comment:
Needs more comments to explain purpose of this type and rest of these changes. It's not clear what/when to use.

Reply:
I will add a package-info.java the explains the pattern and how to use it